doc/whats_new/v1.1.rst

```diff
@@ -124,6 +124,11 @@ Changelog
   backward compatibility, but this alias will be removed in 1.3.
   :pr:`21177` by :user:`Julien Jerphanion <jjerphan>`.
 
+- |API| Parameters ``sample_weight`` and ``multioutput`` of
+  :func:`metrics.mean_absolute_percentage_error` are now keyword-only,
+  in accordance with SLEP009. A deprecation cycle was introduced.
```

Comment:
```suggestion
  in accordance with `SLEP009 <https://scikit-learn-enhancement-proposals.readthedocs.io/en/latest/slep009/proposal.html>`. A deprecation cycle was introduced.
```

A link to the SLEP would be helpful for folks who are not familiar with the process. You can add a link and break the lines accordingly.